drivers/at86rf2xx/include/at86rf2xx_internal.h

```diff
@@ -163,10 +163,18 @@ void at86rf2xx_hardware_reset(at86rf2xx_t *dev);
  */
 void at86rf2xx_configure_phy(at86rf2xx_t *dev);
 
-#if defined MODULE_AT86RF233 || defined MODULE_AT86RF231
+#if defined(MODULE_AT86RF233) || defined(MODULE_AT86RF231) || defined(DOXYGEN)
 /**
  * @brief   Read random data from the RNG
  *
+ * @note    According to the data sheet this function only works properly in
+ *          Basic Operation Mode. However, sporadic testing has shown that even
+ *          in Extended Operation Mode this returns random enough data to be
+ *          used as a seed for @ref sys_random for not relevant for
+ *          cryptography.
```

Comment:
First of all, the current sentence is broken anyway (syntactically).
Second, IMO weak or wrong security is worse than no security.